openmc/deplete/results.py

```diff
@@ -16,6 +16,8 @@
 
 
 def _get_time_as(seconds, units):
+    if units == "y":
+        return seconds / (60 * 60 * 24 * 365.25)  # 365.25 due to the leap year
```

Comment:
I did consider adding year before but I had punted on it because 1) most organizations recommend "a" for the symbol although "y" and "yr" are clearly common, and 2) there is [no universal agreement](https://en.wikipedia.org/wiki/Year#Symbols_and_abbreviations) on what length of time "1 a" actually is. It could be a common year (365), Julian year (365.25), Gregorian year (365.2425), mean tropical year (365.24219). I'm probably being overly pedantic about this though, and from a practical perspective it would be nice for users to have some option for displaying time in years. I would suggest changing the symbol to "a" since there does seem to be agreement on that at least, and then make sure the docstrings make clear which kind of year it refers to.

Reply:
Thanks for the improvements, I've implemented as suggested. So many definitions of years, look forward to adding Mars years one day :earth_africa: